README.md

```diff
@@ -529,28 +531,84 @@ class MyWidget extends WidgetBase<WidgetProperties> {
 }
 ```
 
-### Widget Registry
+### Registry
+
+The `Registry` provides a mechanism to define widgets and injectors (see the [`Containers & Injectors`](#containers--injectors) section), that can be dynamically/lazily loaded on request. Once the registry widget is loaded all widgets that need the newly loaded widget will be invalidated and re-rendered automatically.
+
+A main registry can be provided to the `projector`, which will be automatically passed to all widgets within the tree (referred to as `baseRegistry`). Each widget also gets a access to a private `Registry` instance that can be used to define widget specific registry items. The locally defined registry items are considered a higher precedence than an item registered in the `baseRegistry`.
+
+```ts
+import { Registry } from '@dojo/widget-core/Registry';
+
+import { MyWidget } from './MyWidget';
+import { MyInjector } from './MyInjector';
+
+const registry = new Registry();
+registry.define('my-widget', MyWidget);
+registry.defineInjector('my-injector', MyInjector);
+// ... Mixin and create Projector ...
+
+projector.setProperties({ registry });
+```
+
+In some scenarios, it might be desirable to allow the `baseRegistry` override the an item defined in the local `registry`. To do this use pass `true` as the second argument of the `registry.get` function.
```

Comment:
To do this use pass `true` as the second argument of the `registry.get` function. -> Use `true` as the second argument of the `registry.get` function to override the local item.